lib/iex/resources/symbols.rb

```diff
@@ -0,0 +1,17 @@
+module IEX
+  module Resources
+    class Symbols < Resource
+      property 'symbol'
+      property 'exchange'
+      property 'name'
+      property 'date'
+      property 'is_enabled', from: 'isEnabled'
```

Comment:
I'm not sure, but maybe it would be good to have `enabled?` property or method to be more ruby style

Reply:
To be honest, I was worried about that too.
However, if you change from the original name, it would be hard to tell the difference from the original name, so I made it that way.

Reply:
I would call this `enabled` and define `enabled?` as an additional method.